xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-test/xwiki-platform-livedata-test-pageobjects/src/main/java/org/xwiki/livedata/test/po/LiveDataElement.java

```diff
@@ -144,6 +144,22 @@ public LiveDataElement setPagination(int paginationNumber)
         return this;
     }
 
+    /**
+     *
+     * @return the possible pagination sizes.
+     * @since 16.5.0
+     */
+    public List<Integer> getPaginationPageSizes()
+    {
+        WebElement element = getRootElement().findElement(By.cssSelector(".pagination-page-size select"));
+        return new Select(element)
+            .getOptions()
+            .stream()
+            .map(WebElement::getText)
+            .map(Integer::parseInt)
+            .collect(Collectors.toList());
```

Comment:
Available since Java 16:
```suggestion
            .toList();
```